### PR TITLE
Update RDPApps.reg link

### PR DESCRIPTION
**Pull request by Leonardo-DGS**

### CODE REVIEW

**Comment by oskardotglobal** [comment]
Thank you for your contribution. However, we can't merge this yet due to legal issues we're currently resolving.

File: docs/KVM.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ Leave everything as default and click `Next` through the installer. This will in
 
 ![](kvm/25.png)
 
-Once you finish the driver install, you will need to make some registry changes to enable RDP Applications to run on the system. Start by downloading the `RDPApps.reg` file from the WinApps repo by visiting https://github.com/Fmstrat/winapps/blob/main/install/RDPApps.reg, right clicking on the `Raw` button, and clicking on `Save target as`.
+Once you finish the driver install, you will need to make some registry changes to enable RDP Applications to run on the system. Start by downloading the `RDPApps.reg` file from the WinApps repo by visiting https://github.com/winapps-org/winapps/blob/main/install/RDPApps.reg, right clicking on the `Raw` button, and clicking on `Save target as`.
```
Comment (fbartels) [inline docs/KVM.md:138]: Can we make this link relative?

**Review by fbartels — CHANGES_REQUESTED** [review]

**Review by oskardotglobal — APPROVED** [review]
Looks good

**Comment by LDprg** [comment]
Sure a relative link would be copied correctly when right clicking -> Copy? Because many people just copy the link that way and past it into the vm.

**Review by fbartels — APPROVED** [review]

**Comment by fbartels** [comment]
@LDprg no, this would not work for a direct paste into curl. But the behaviour with a relative or absolute link still give the same result (a Github page). The readme also explains what do it. So while not optimal, i think there is enought guidance on what to do.

**Comment by Leonardo-DGS** [comment]
> Sure a relative link would be copied correctly when right clicking -> Copy? Because many people just copy the link that way and past it into the vm.

I've tried and it works.

**Comment by oskardotglobal** [comment]
Can confirm. Merging this since relicensing isn't an issue, see: https://github.com/winapps-org/winapps/discussions/11#discussioncomment-6461682

**Comment by oskardotglobal** [comment]
@Leonardo-DGS Thank you for your contribution

**Pull request merged**